java/src/main/java/vowpalWabbit/VW.java

```diff
@@ -16,8 +16,9 @@ private VW(){}
      * This main method only exists to test the library implementation.  To test it just run
      * java -cp target/vw-jni-*-SNAPSHOT.jar vowpalWabbit.VW
      * @param args No args needed.
+     * @throws Exception possibly during close.
      */
-    public static void main(String[] args) {
+    public static void main(String[] args) throws Exception {
```

Comment:
I don't know if all of these are required.  Might just need to be an `IOException`.